### ISSUE TITLE
Log spam

### ISSUE THREAD

**Issue by BioTechproject27** [Minenash/RebindAllTheKeys#29]
When "intentional crash" is unbound,
logs are spammed with
```
[Render thread/ERROR]:
 ########## GL ERROR ##########
[Render thread/ERROR]:
 @ Render
[Render thread/ERROR]:
 65539: Invalid key 1
 ```
 when pressing any key.
 It is successfully unbound, it won't crash, however f3+c still copies location to clipboard (since it's kinda 2 features in 1).

**Comment by BioTechproject27** [comment]
Oh yeah, found in 1.20.1 using version 1.3.0+1.20.1

**Comment by YoshiRulz** [comment]
Still present in 1.4.0. Possibly related to LynixPlayz/F3Keybinding#1. Workaround is to set it to `key.keyboard.f25` since that's valid for some reason and will never be used.

edit: Also happens when pressing `Debug Key` if `Cycle Render Distance` is unbound.

**Comment by Minenash** [comment]
This issue is annoying since I don't really know how it actually happens, so it's hard to fix the error itself. That being said, I know *why* this issues occurs (I used negative key numbers to avoid getting the duplicate key thing), and I should change to a different way of handling that, especially now that mojang has a newer system.

**Comment by Minenash** [comment]
Fixed in next release

**Comment by Minenash** [comment]
Fixed in Version 1.5.0. [Modrinth](<https://modrinth.com/mod/rebind-all-the-keys/version/1.5.0+1.20.2>) [Curseforge](<https://www.curseforge.com/minecraft/mc-mods/rebindallthekeys/files/5242211>) [Github](<https://github.com/Minenash/RebindAllTheKeys/releases/tag/v1.5.0%2B1.20.2>)
